src/drive/web/modules/upload/UploadButton.jsx

```diff
@@ -2,51 +2,25 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { compose } from 'redux'
-import { Icon } from 'cozy-ui/react'
+import { Icon, FileInput } from 'cozy-ui/react'
 
 import withSharingState from 'sharing/hoc/withSharingState'
 import { uploadFiles } from 'drive/web/modules/navigation/duck'
 
-const styles = {
-  parent: {
-    position: 'relative',
-    overflow: 'hidden'
-  },
-  input: {
-    position: 'absolute',
-    top: 0,
-    left: 0,
-    opacity: 0,
-    width: '100%',
-    height: '100%',
-    zIndex: 1
-  }
-}
-
 const UploadButton = ({ label, disabled, onUpload, className }) => (
-  <label
-    role="button"
-    disabled={disabled}
+  <FileInput
     className={className}
-    style={styles.parent}
+    data-test-id="upload-btn"
+    label={label}
```

Comment:
Seeing this a bit late, but I guess a `label` prop makes it sound like a label will be visually rendered when it's about a11y. Maybe `aria-label` would have been better?

Reply:
I agree. I will make a PR in cozy-ui

Reply:
well in fact, there is no `label` prop in `FileInput`, thanks :-)

Reply:
There is no `label` prop in `FileInput` component in fact. So the label we pass here is just passed to the hidden input so I don't think we need this in the end.